### PR TITLE
[Task] Add display list of songs & Navigation controller

### PR DESCRIPTION
**Pull request by saske999cz**
## Related Tickets
- [#Task xxxxx: Content](https://edu-redmine.sun-asterisk.vn/issues/xxxx)
## WHAT
- Add 

display list of songs & Navigation controller
## Evidence (Screenshot or Video)
Screenshot 2023-07-05 at 14 45 58
Screenshot 2023-07-05 at 14 46 07
## Review Checklist
Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/blob/master/eng/swift/coding_convention.md | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*
*(Other notes)*

### CODE REVIEW

File: App_Music/CustomSongCell.swift [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import UIKit
+
+class CustomSongCell: UITableViewCell {
```
Comment (tunghs-1253) [inline App_Music/CustomSongCell.swift]: final

File: App_Music/CustomSongCell.swift [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    @IBOutlet weak var songName: UILabel!
+    @IBOutlet weak var songArtist: UILabel!
+    @IBOutlet weak var songImage: UIImageView!
```
Comment (tunghs-1253) [inline App_Music/CustomSongCell.swift]: private

File: App_Music/CustomSongCell.swift [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+
+
```
Comment (tunghs-1253) [inline App_Music/CustomSongCell.swift]: Dòng trống

File: App_Music/CustomSongCell.swift [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import UIKit
+
+class CustomSongCell: UITableViewCell {
+    @IBOutlet weak var songName: UILabel!
+    @IBOutlet weak var songArtist: UILabel!
+    @IBOutlet weak var songImage: UIImageView!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
```
Comment (tunghs-1253) [inline App_Music/CustomSongCell.swift]: Dòng trống

File: App_Music/LibraryVC.swift [hunk 5]
```diff
@@ -0,0 +1,74 @@
+import UIKit
+
+class LibraryVC: UIViewController {
```
Comment (tunghs-1253) [inline App_Music/LibraryVC.swift]: final

File: App_Music/LibraryVC.swift [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+    @IBOutlet private weak var songName: UILabel!
+    @IBOutlet private weak var songArtist: UILabel!
+    @IBOutlet private weak var songImage: UIImageView!
+    private var songLibrary: [Song] = []
```
Comment (tunghs-1253) [inline App_Music/LibraryVC.swift]:
```suggestion
    @IBOutlet private weak var songImage: UIImageView!
    
    private var songLibrary: [Song] = []
```

Comment (tunghs-1253) [inline App_Music/LibraryVC.swift]:
```suggestion
    @IBOutlet private weak var songImage: UIImageView!
    
    private var songs: [Song] = []
```

File: App_Music/LibraryVC.swift [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+    private var toSongVCIdentifier: String = Constants.toSongVCSegue.rawValue
+    private var jpg: String = Constants.jpg.rawValue
```
Comment (tunghs-1253) [inline App_Music/LibraryVC.swift]:
```suggestion
    private var jpgRawValue: String = Constants.jpg.rawValue
```

File: App_Music/LibraryVC.swift [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+    
+
+
+}
+
+
+extension LibraryVC: UITableViewDelegate, UITableViewDataSource {
+    
+     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 10
```
Comment (tunghs-1253) [inline App_Music/LibraryVC.swift]: magic number

File: App_Music/Constants.swift [hunk 9]
```diff
@@ -5,4 +5,5 @@ enum Constants: String{
     case lastSongIndex    = "LastSongIndex"
     case mp3              = "mp3"
     case jpg              = "jpg"
-}
\ No newline at end of file
+    case toSongVCSegue = "ToSongVC"
```
Comment (ghost) [inline App_Music/Constants.swift]: space thêm tí cho đẹp á Thái

**Review by ghost — APPROVED** [review]

**Pull request merged**